receiver/mongodbatlasreceiver/README.md

```diff
@@ -32,6 +36,14 @@ below both values are being pulled from the environment.
   - `tls`
     - `key_file`
     - `cert_file`
+- `logs`
+  - `enabled` (default true)
```

Comment:
I don't think this should be enabled by default, particularly because additional configuration is required. It's also a breaking change.